vmdb/spec/automation/unit/method_validation/unregister_from_provider_spec.rb

```diff
@@ -1,32 +0,0 @@
-require File.expand_path(File.join(File.dirname(__FILE__), '..', '..',
```

Comment:
@tinaafitz 
Since we have unregister as a valid instance now can we revive this spec and change the name of the instance.